.ado/variables/vs2019.yml

```diff
@@ -1,8 +1,7 @@
 variables:
   VmImage: windows-2019
   AgentPool.Medium: rnw-pool-4
-  AgentPool.Interactive: rnw-pool-4-ui
-  AgentPool.Large: windevbuildagents
+  AgentPool.Large: rnw-pool-8
   MSBuildVersion: 16.0
   BaseIntDir: $(Agent.HomeDirectory)\BaseIntDir # redirect to C:
```

Comment:
Is this override still needed?

Reply:
GH is very bad at pointing which line you are referring to, I'm assuming `BaseIntDir`. 
I say we should be able to get rid of that I suspect that was to deal with disk size issues on the smaller vm's.
@NickGerleman to double-check if I'm correct.
But let's do in seperate PR.

Reply:
Yeah, @licanhua added the redirection a ways back when we had less disk space and needed to install multiple versions of VS at once. It should be safe to remove, and probably preferable since we've had issues before where our default version was incorrect.

Reply:
Yeah, I meant this one:

```
BaseIntDir: $(Agent.HomeDirectory)\BaseIntDir # redirect to C:
```

Just wondering if the "hack" is even needed anymore. Sorry to digress.